react/package.json

```diff
@@ -30,15 +36,19 @@
   },
   "jest": {
     "verbose": true,
-    "testURL": "http://localhost/",
     "transform": {
+      "\\.(gql|graphql)$": "jest-transform-graphql",
```

Comment:
Do you know why is this necessary, since this app apparently doesn't use `.gql`?

Reply:
Dunno. I will remove and test to see if there is any colateral effect.